ubuntu/common/disable_auto_upgrade.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+KERNEL_VERSION=$(uname -r) su -c 'echo linux-image-$KERNEL_VERSION hold | dpkg --set-selections'
+sed -i 's/APT::Periodic::Unattended-Upgrade ".*/APT::Periodic::Unattended-Upgrade "0";/' /etc/apt/apt.conf.d/20auto-upgrades
+
+# Holding on the auto updates for nvidia fabric manager and cuda drivers
+apt-mark hold nvidia-fabricmanager-* cuda-drivers-fabricmanager-*
```

Comment:
Please move this immediately after the component installation, and use specific version.

What is cuda-drivers-fabricmanager?

Reply:
the cuda packages present were from nvidia-fabricmanager
specifically they were cuda-drivers-fabricmanager-450